katcp/sampling.py

```diff
@@ -92,6 +92,17 @@ class SampleStrategy(object):
         DIFFERENTIAL_RATE: b"differential-rate",
     }
 
+    ## @brief Mapping from strategy constant to number of parameters.
+    SAMPLING_NUM_PARAMS_LOOKUP = {
+        NONE: 0,
+        AUTO: 0,
+        PERIOD: 1,
+        EVENT: 0,
+        DIFFERENTIAL: 1,
+        EVENT_RATE: 2,
+        DIFFERENTIAL_RATE: 3,
+    }
```

Comment:
We should probably fix this, since we are delimiting by `,` instead of ` `

Reply:
This dict is no longer required, and has been removed.